server/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "wire-webapp-server",
```

Comment:
Lets call it `wire-web-server` because it will be used for other projects as well